common-test/src/main/scala/org/scalatest/SharedHelpers.scala

```diff
@@ -828,7 +829,7 @@ object SharedHelpers extends Assertions with LineNumberHelper {
         case map: GenMap[_, _] => element.asInstanceOf[Tuple2[_, _]]._1
         case genTrv: GenTraversable[_] => getIndex(xs, element)
       }
-    Array(indexOrKey.toString, decorateToStringValue(element))
+    Array(indexOrKey.toString, decorateToStringValue(Prettifier.default, element))
```

Comment:
Why is Prettifier.default used here? Why isn't a captured Prettifier passed down?